### ISSUE TITLE
ssh-remote install is not working

### ISSUE THREAD

**Issue by kino71** [microsoft/vscode-remote-release#2736]
- VSCode Version: 1.44.0
- Local OS Version: windows 10
- Remote OS Version: Red Hat 8
- Remote Extension/Connection Type: SSH

Access to the host is through cyberark. I can successfully open a ssh terminal within vscode
I tried both windows ssh.exe and PortableGit's

Steps to Reproduce:

Upon first ssh-remote connect I get the following error:

[10:07:06.149] SSH Resolver called for "ssh-remote+7b22686f73744e616d65223a224445564c494e5558227d", attempt 1
[10:07:06.149] SSH Resolver called for host: DEVLINUX
[10:07:06.149] Setting up SSH remote "DEVLINUX"
[10:07:06.168] Using commit id "2aae1f26c72891c399f860409176fe435a154b13" and quality "stable" for server
[10:07:06.170] Install and start server if needed
[10:07:09.903] Checking ssh with "C:\Apps\PortableGit\usr\bin\ssh.exe -V"
[10:07:10.019] > OpenSSH_8.2p1, OpenSSL 1.1.1e  17 Mar 2020
[10:07:10.021] Running script with connection command: "C:\Apps\PortableGit\usr\bin\ssh.exe" -T -D 64699 DEVLINUX bash
[10:07:10.024] Terminal shell path: C:\Windows\System32\cmd.exe
[10:07:10.355] > 
[10:07:10.356] Got some output, clearing connection timeout
[10:07:10.991] > [username]@[username_cyberark]@test-rh8@[cyberark_host.domain]'s password:
[10:07:31.776] > 
> 
[10:07:33.396] > 
> This session is being recorded
> 
[10:07:36.629] > 5be7f6df47e2: running
> 
[10:07:36.668] > bash: line 193: unexpected EOF while looking for matching `"'
> bash: line 194: syntax error: unexpected end of file
>

**Comment by roblourens** [comment]
It seems like maybe the install script that we pipe into ssh is being truncated? It should be longer than 194 lines and there is nothing special about those lines...

Anything different if you set `"remote.SSH.useLocalServer": true`? This is just a different install method that may work around whatever is going wrong here.

**Comment by kino71** [comment]
I tried to play with useLocalServer with no luck.
It seems that CyberArk that act as a ssh proxy, do something that breaks the installation process. I tried the same host through CyberArk and directly, the former method shows the issue, the latter doesn't.

Unfortunately CyberArk will soon be mandatory on all hosts in my organisation soon, I'll have to work on that with my colleagues who take care of that product.

Thanks!
